Description: bump iceweasel max version
 Bump Iceweasel maximum version to 24.*, as it's fully supported
 .
 xul-ext-image-tools (1.0.7-1nano) unstable; urgency=low
 .
   * initial release
Author: Christopher Roy Bratusek <nano@tuxfamily.org>

--- xul-ext-image-tools-1.0.7.orig/install.rdf
+++ xul-ext-image-tools-1.0.7/install.rdf
@@ -17,10 +17,10 @@
 			<Description>
 				<em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
 				<em:minVersion>3.6</em:minVersion>
-				<em:maxVersion>4.0.*</em:maxVersion>
+				<em:maxVersion>24.*</em:maxVersion>
 			</Description>
 		</em:targetApplication>
-		
+
 		<em:iconURL>chrome://imagetools/skin/icon.png</em:iconURL>
 	</Description>
 </RDF>
